File: exemplos/java/TabelaHash.java

package br.edu.ifgoiano.exemplo01;

import java.util.HashMap;
import java.util.Map;

/**
 *
 * @author profronneesley
 */
public class TabelaHash {

    public static void main(String[] args) {
        Map<String, Double> mapa = new HashMap<>();
        
        mapa.put("ana", 10.0);
        mapa.put("roni", 8D);
        mapa.put("pedro", 9D);
        
        for (String chave: mapa.keySet()){
            System.out.println(chave + 
                    " nota: " + mapa.get(chave));
        }
    }
}

File: exemplos/julia/tabela_hash.jl
julia> mapa = Dict("roni" => 8, "ana" => 10, "bob" => 3)
Dict{String, Int64} with 3 entries:
  "ana"  => 10
  "bob"  => 3
  "roni" => 8

julia> mapa["pedro"] = 9
9

julia> mapa
Dict{String, Int64} with 4 entries:
  "ana"   => 10
  "pedro" => 9
  "bob"   => 3
  "roni"  => 8
